ghost/i18n/locales/sl/portal.json

```diff
@@ -1,212 +1,212 @@
+    "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Če je bila pritožba zaradi vsiljene pošte naključna ali želite ponovno prejemati e-pošto, se lahko ponovno prijavite na e-pošto z gumbom na prejšnjem zaslonu.",
+    "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Če zdaj prekličete naročnino, boste imeli še vedno dostop do {{periodEnd}}.",
+    "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Če imate poslovni ali vladni e-poštni račun, se obrnite na svoj IT oddelek in jih prosite, naj omogočijo prejemanje e-pošte iz {{senderEmail}}.",
+    "If you would like to start receiving emails again, the best next steps are to check your email address on file for any issues and then click resubscribe on the previous screen.": "Če želite ponovno prejemati e-pošto, so najboljši naslednji koraki, da preverite svoj e-poštni naslov za morebitne težave in se nato ponovno naročite na prejšnjem zaslonu.",
+    "If you're not receiving the email newsletter you've subscribed to, here are a few things to check.": "Če ne prejemate novic, na katere ste se naročili, je tukaj nekaj stvari, ki jih preverite.",
+    "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Če ste izvedli vse te preglede in še vedno ne prejemate e-pošte, se lahko obrnete na podporo prek {{supportAddress}}.",
+    "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "Če pri pošiljanju e-novic pride do trajne napake, bo e-pošta na računu onemogočena.",
+    "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "V svojem e-poštnem odjemalcu dodajte {{senderEmail}} na seznam stikov. To signalizira vašemu ponudniku e-pošte, da je e-pošta, poslana s tega naslova, zaupanja vredna.",
+    "Invalid email address": "Neveljavna e-pošta",
+    "Jamie Larson": "Janez Novak",
+    "jamie@example.com": "janez@primer.si",
+    "Less like this": "Manj takšnih vsebin",
+    "Make sure emails aren't accidentally ending up in the Spam or Promotions folders of your inbox. If they are, click on \"Mark as not spam\" and/or \"Move to inbox\".": "Prepričajte se, da e-pošta ne konča v mapi za vsiljeno pošto ali promocije. Če se to zgodi, kliknite na \"Ni vsiljena pošta\" and/or \"Premakni v nabiralnik\".",
```

Comment:
Looks like you missed translating "and" and "or" here.  (English uses and/or when both or one might be true.  If there's an alternative way to say this in Slovenian, that's fine, but please do translate.)